common/constants/validations.js

```diff
@@ -1,2 +1,2 @@
 export const minimumPasswordLength = 8;
-export const passwordStrengthRegex = /^(?=.*[a-z])(?=.*[A-Z])(?=.*\d).+/;
+export const passwordStrengthRegex = /^(?=.*[a-z])(?=.*[A-Z])(?=.*\d).{8,}/;
```

Comment:
this isn't necessary, because we test length via the variable above (connected to `Yup.min()`).

Reply:
I had taken the Yup.min() check off as it could all be taken care of via the regex. However, I can put it back on!

Reply:
Yes please! I'd prefer it the other way around